server/middleware/fetchFrontendComponentMiddleware.ts

```diff
@@ -0,0 +1,21 @@
+import { RequestHandler } from 'express'
+import type { FrontendComponentsPageAdditions } from 'viewModels'
+import logger from '../../logger'
+import { Services } from '../services'
+
+export default function getFrontendComponents({ frontendComponentService }: Services): RequestHandler {
+  return async (req, res, next) => {
+    try {
+      const footer = await frontendComponentService.getComponents('footer', res.locals.user.token)
+      res.locals.feComponents = {
+        footerHtml: footer.html,
+        cssIncludes: [...footer.css],
+        jsIncludes: [...footer.javascript],
+      } as FrontendComponentsPageAdditions
+      next()
+    } catch (error) {
+      logger.error(error, 'Failed to retrieve front end components')
```

Comment:
I'm not sure this is ever triggered as I _think_ you're catching the error in the service (which is where I think you should be catching it)
Its probably OK to merge at the moment (to allow you to move forward), but I think I'll look at this once you've merged 👍 👀